docs/migrating/backup.md

```diff
@@ -0,0 +1,59 @@
+# Backup 
+
+You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux/UNIX based system where eZ Platform is using a MySQL database called "example". You should close the system before making a backup or an upgrade.
+
+!!! note "Externally stored assets"
+
+    If you store assets in any external service or localization, you should back them up before proceeding.
+
+1\. Navigate into the eZ Platform directory:
+ 
+```
+$ cd /path/to/ezplatform
```

Comment:
We don't use the `$` before bash commands elsewhere, let's remove it here.
Also applies below